src/Umbraco.Core/Notifications/ContentPublishedNotification.cs

```diff
@@ -21,8 +21,11 @@ public ContentPublishedNotification(IEnumerable<IContent> target, EventMessages
     {
     }
 
-    /// <summary>
-    /// Gets a enumeration of <see cref="IContent"/> with the published entities.
+    public ContentPublishedNotification(IEnumerable<IContent> target, EventMessages messages, bool includeDescendants)
+        : base(target, messages) => IncludeDescendants = includeDescendants;
+
     /// </summary>
     public IEnumerable<IContent> PublishedEntities => Target;
+
+    public bool IncludeDescendants { get; set; }
```

Comment:
@nul800sebastiaan We should remove the property setter here, because this shouldn't be a mutable value (handlers can now change the value) ⚠️

Reply:
Thanks @ronaldbarendse - that's updated in https://github.com/umbraco/Umbraco-CMS/commit/bc127d95bab195702a630bd1cd3f18fc6a13ed1a